app/controllers/api/v1/jobs_controller.rb

```diff
@@ -1,7 +1,8 @@
 class Api::V1::JobsController < ApplicationController
 
   def index
-    paginate json: Job.by_date, per_page: 25
+    jobs = params[:technology].nil? ? Job.by_date : Job.by_date.by_tech(params[:technology])
```

Comment:
Yeah, we may want to reconsider how it's written if we add another query type... make it more dynamic? But who am I kidding...it won't be "we" ;)